api/lib/domain/usecases/disable-certification-center-membership-from-pix-certif.js

```diff
@@ -0,0 +1,41 @@
+import { NotFoundError } from '../errors.js';
+import { ForbiddenError } from '../../application/http-errors.js';
+
+const disableCertificationCenterMembershipFromPixCertif = async function ({
+  certificationCenterMembershipId,
+  updatedByUserId,
+  certificationCenterMembershipRepository,
+}) {
+  const membershipToDisable = await certificationCenterMembershipRepository.findById(certificationCenterMembershipId);
+
+  if (!membershipToDisable) {
+    throw new NotFoundError(`Cannot find a certification center membership for id ${certificationCenterMembershipId}`);
+  }
+
+  const membershipCanBeDisabled = await _membershipCanBeDisabled({
+    membershipToDisable,
+    certificationCenterMembershipRepository,
+  });
+
+  if (!membershipCanBeDisabled) {
+    throw new ForbiddenError(`Cannot disabled membership with id ${certificationCenterMembershipId}`);
+  }
+
+  membershipToDisable.disableMembership(updatedByUserId);
+
+  certificationCenterMembershipRepository.update(membershipToDisable);
+};
+
+async function _membershipCanBeDisabled({ membershipToDisable, certificationCenterMembershipRepository }) {
+  if (!membershipToDisable.hasAdminRole) {
+    return true;
+  }
+
+  const currentActiveAdmins = await certificationCenterMembershipRepository.findActiveAdminsByCertificationCenterId(
+    membershipToDisable.certificationCenter?.id,
```

Comment:
Dans quel cas un membership pourrait-il ne pas avoir de certification center ? Aussi je propose de remplacer `?.` par `.` : 

```suggestion
    membershipToDisable.certificationCenter.id,
```